third_party/freetype2/.gitmodules

```diff
@@ -1,3 +0,0 @@
-[submodule "dlg"]
```

Comment:
Why do we delete this file? Don't we need it for subprojects/dlg?

Reply:
I think it was because we were having .gitmodule errors where git was ignoring the gitmodules = false setting, but @kaidokert has a fix for that.

I don't think we should delete .gitmodules if we don't need to in order to keep Cobalt changes minimal.

Reply:
We don't need to delete it, and yes we _should_ keep it minimal. In fact it the root cause turned up not to be related to the `.gitmodules` file at all, the issue was in github actions code.